### PR TITLE
Improve responses in `invoke local`

### LINKED ISSUE #2620: Improve responses in `invoke local`
# This is a proposal

## Description

When invoking a function remotely via `serverless invoke`, it returns ONLY the response, whether that be an error or a success message.

When invoking a local function via `invoke local`, it returns things differently:

* On success, `invoke local` returns the response, but right before that it prints:
```
Serverless: successfully invoked your function
```  
This should be removed.  The first reason is that it's obvious whether it has succeeded or failed.  The second reason is that there are a lot of plugins that want to parse the response from the `invoke` command, but if we put Serverless statements in that response, it makes it really hard to parse, since the CLI standard output has to be parsed and the log statement is mixed in with the that.

* On error, it returns an error that looks like something went wrong with the Serverless Framework, not the user's code:

```
$ sls invoke local -f connect
 
  Syntax Error -------------------------------------------
 
     Unexpected token u in JSON at position 0
 
     For debugging logs, run again after setting SLS_DEBUG env var.
 
  Get Support --------------------------------------------
     Docs:          docs.serverless.com
     Bugs:          github.com/serverless/serverless/issues
 
     Please report this error. We think it might be a bug.
 
  Your Environment Information -----------------------------
     OS:                 darwin
     Node Version:       6.6.0
     Serverless Version: 1.1.0
```

This isn't the right experience.  We should catch the error and return a specific looking error that explains the code is wrong, the line, stacktrace, etc...

### PR DESCRIPTION
**Pull request by horike37**
## What did you implement:

Closes #2620 

## How did you implement it:
- On success, invoke local returns only the response. Removed the message `Serverless: successfully invoked your function`.

- On error, it returns the code is wrong, the line, stacktrace as  follow
```
$../serverless/bin/serverless invoke local --function function1
/Users/horike/src/develop-serverless/src/function1/handler.js:7
  sss{
     ^
SyntaxError: Unexpected token {
    at Object.exports.runInThisContext (vm.js:76:16)
    at Module._compile (module.js:528:28)
    at Object.Module._extensions..js (module.js:565:10)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (module.js:483:17)
    at require (internal/module.js:20:19)
    at AwsInvokeLocal.invokeLocalNodeJs (/Users/horike/src/serverless/lib/plugins/aws/invokeLocal/index.js:68:16)
    at AwsInvokeLocal.invokeLocal (/Users/horike/src/serverless/lib/plugins/aws/invokeLocal/index.js:50:19)
    at AwsInvokeLocal.tryCatcher (/Users/horike/src/serverless/node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (/Users/horike/src/serverless/node_modules/bluebird/js/release/promise.js:510:31)
    at Promise._settlePromise (/Users/horike/src/serverless/node_modules/bluebird/js/release/promise.js:567:18)
    at Promise._settlePromise0 (/Users/horike/src/serverless/node_modules/bluebird/js/release/promise.js:612:10)
    at Promise._settlePromises (/Users/horike/src/serverless/node_modules/bluebird/js/release/promise.js:691:18)
    at Async._drainQueue (/Users/horike/src/serverless/node_modules/bluebird/js/release/async.js:138:16)
    at Async._drainQueues (/Users/horike/src/serverless/node_modules/bluebird/js/release/async.js:148:10)
    at Immediate.Async.drainQueues (/Users/horike/src/serverless/node_modules/bluebird/js/release/async.js:17:14)
    at runCallback (timers.js:574:20)
    at tryOnImmediate (timers.js:554:5)
    at processImmediate [as _immediateCallback] (timers.js:533:5)

```

## Todos:

- [ ] Write tests
- [ ] Write documentation
- [x] Fix linting errors
- [x] Make sure code coverage hasn't dropped
- [x] Provide verification config/commands/resources
- [x] Enable ["Allow edits from maintainers"](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) for this PR
- [x] Change ready for review message below

***Is this ready for review?:*** YES

### CODE REVIEW

**Comment by ac360** [comment]
Yes!  Thanks for doing this @horike37.  Really great job :)  I haven't reviewed the code, but this should definitely get in before next week's release. cc @flomotlik @eahefnawy

**Comment by eahefnawy** [comment]
Looks great! G2M 🙌 

Thanks a lot @horike37 ☺️
